src/upgrade.rs

```diff
@@ -273,7 +273,7 @@ pub async fn get_upgrades<'a, 'b>(
         );
     }
 
-    let input = input(config, "Packages to exclude: ");
+    let input = input(config, "Packages to exclude: (eg: \"1 2 3\", \"1-3\")");
```

Comment:
Here you've quoted "1 2 3" while the search install has it unquoted.

Reply:
OK. Did as in the search. thanks for review!

Reply:
There's also differences in the `:` placement.  I don't really know what I prefer, just they should be the same.

Reply:
Hmmm, the semicolon at the end looks better.

Reply:
Looks good now.